charts/mi-extraction-service/values.yaml

```diff
@@ -1,3 +1,29 @@
-java:
+mi-job:
+  image: ${IMAGE_NAME}
+  memoryLimits: '2048Mi'
+  cpuLimits: '2000m'
+  activeDeadlineSeconds: 3600
+  aadIdentityName: mi-extraction-service
+  kind: Job
+  labels:
+    app.kubernetes.io/instance : mi-extraction-service-build-001
+    app.kubernetes.io/name: mi-extraction-service-build
   environment:
-    image: 'ssprivatesbox.azurecr.io/mi/mi-extraction-service:latest'
+    STORAGE_STAGING_NAME: mipersistent{{ .Values.global.environment }}
+    STORAGE_EXPORT_NAME: miexport{{ .Values.global.environment }}
+    MAIL_TARGETS: tony.chow@hmcts.net
```

Comment:
probably you don't want to merge this

Reply:
Yep :+1:, I'll make it a secret instead. Easier when moving to GovUK Notify and I'll use the secret as an API Key isntead.